pkg/funky/mocks/Server.go

```diff
@@ -0,0 +1,128 @@
+// Code generated by mockery v1.0.0
```

Comment:
File names should have only lowercase letters and underscores

Reply:
oh that's right, there is an option in mockery for that.